packages/contracts/test/voting/token-voting.ts

```diff
@@ -756,16 +767,19 @@ describe('TokenVoting', function () {
         await advanceIntoVoteTime(startDate, endDate);
 
         // vote with 50 yes votes
-        await governanceErc20Mock.mock.getPastVotes.returns(50);
-        await voting.vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[0]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[1]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[2]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[3]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[4]).vote(id, VoteOption.Yes, false);
 
-        // vote 30 voting no votes
-        await governanceErc20Mock.mock.getPastVotes.returns(30);
-        await voting.connect(signers[1]).vote(id, VoteOption.No, false);
+        // vote with 30 no votes
+        await voting.connect(signers[5]).vote(id, VoteOption.No, false);
+        await voting.connect(signers[6]).vote(id, VoteOption.No, false);
+        await voting.connect(signers[7]).vote(id, VoteOption.No, false);
```

Comment:
could also be optimized with a loop and `Promise.all`

Reply:
Improved in e66e59747b75c6ce535bde30be1342d74a358978